package.json

```diff
@@ -28,7 +28,7 @@
     "serve.prod": "gulp serve.prod --color",
     "start": "gulp serve.dev --color",
     "tasks.list": "gulp --tasks-simple --color",
-    "test": "gulp test --color",
+    "test": "gulp test --color && gulp build.prod --color && gulp build.js.e2e --color && gulp e2e --color",
```

Comment:
Lets move `gulp e2e` to a script called `e2e` and thus:

```bash
npm run e2e
```

Most developers will expect to run only karma when run `npm t`.